packages/gatsby/cache-dir/loader.js

```diff
@@ -167,12 +187,28 @@ const queue = {
   addProdRequires: prodRequires => {
     asyncRequires = prodRequires
   },
-
+  addDataPaths: dataPaths => {
+    jsonDataPaths = dataPaths
+  },
   dequeue: () => resourcesArray.pop(),
-  enqueue: rawPath => {
+  enqueue: async rawPath => {
```

Comment:
we haven't been using async/await as the polyfills are super heavy

Reply:
Which is tragic 😢

Reply:
Right, callbacks it is then!